### PR TITLE
Benine api connections

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Routes for getting presets and/or cameras from thebenine API, post routes for changing settings like coupled camera's and presets will follow tomorrow

### CODE REVIEW

**Comment by Markvanderuit** [comment]
No comments on the code. Looks fine, so we can merge IMO.

**Comment by martijnjanssen** [comment]
I would change the routes to a format without the `-`. The route `/cameras` and `/presets` would be a more logical choice in my opinion.

File: routes/benine.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// Get presets data from benine
+router.get("/get-presets", (req, res) => {
+    const benineHelper = new BenineHelper();
+    benineHelper.getPresets((presets) => {
+        res.json({ presets });
+    });
+});
+
+// Get presets data from benine for a certain camera
+router.get("/get-presets/:id(\\d+)", (req, res) => {
```
Comment (martijnjanssen) [inline routes/benine.js]: Is this id the id of the camera? If so, please change the parameter name to `cameraId`. It looks like the current format gets the preset according to the id of the preset you are getting. An even more logical choice would be to change this route to `/cameras/{id}/presets`.

**Comment by martijnjanssen** [comment]
Looks good!

**Pull request merged**